yarn.lock

```diff
@@ -8893,7 +8861,7 @@ react-popper@^2.1.0:
 
 react-select@3.0.8:
   version "3.0.8"
-  resolved "https://registry.yarnpkg.com/react-select/-/react-select-3.0.8.tgz#06ff764e29db843bcec439ef13e196865242e0c1"
+  resolved "https://registry.yarnpkg.com/react-select/-/react-select-3.0.8.tgz#1c002f04997401a11bede0852fea0807c6aebe41"
```

Comment:
okay this this is the new correct SHA that you replaced which allowed it to actually lower react-select down to 3.0.8?